client/pages/Login/Login.scss

```diff
@@ -0,0 +1,69 @@
+.login_container {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  width: 100%;
+  min-height: 100vh;
+  background-image: url('http://res.cloudinary.com/dd3lv0o93/image/upload/v1531318285/landing_page.png');
+  background-color: rgba(0,0,0,.77);
+  background-blend-mode: multiply;
+  background-size: cover;
+
+  .login_container__section {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 1076px;
+    max-height: 596px;
+    width: 100%;
+    margin: auto;
+    padding: 0 30px;
+
+    p {
+      font-size: 44px;
+      font-weight: 300;
+      text-align: center;
+      color: #FFF;
+
+      span {
+        color: #FFA300;
+      }
+    }
+
+    .login_container__btn {
+      background-color: #FFA300;
+      padding: 21px 24px;
+      font-size: 20px;
+      text-decoration: none;
+      color: #000;
+      cursor: pointer;
+    }
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .login_container {
+    padding: 20px 0;
+    -moz-background-position: 50%;
+    -webkit-background-position: 50%;
+    background-position: 50%;
+
+    .login_container__section {
+      line-height: 1.5px;
+
+      image {
+        width: 100px;
+        height: 100px;
+      }
+  
+      p {
+        font-size: 40px;
+
+        &:nth-child(3) {
+          font-size: 20px;
```

Comment:
we should have a typography.scss, were we define font sizes like this and remember to stick with `em` or `rem`